lib/register_sources_bods/services/es_index_creator.rb

```diff
@@ -17,6 +17,13 @@ def initialize(
         @index = index
       end
 
+      def create_index_unless_exists
+        create_index
+      rescue Elasticsearch::Transport::Transport::Errors::BadRequest => e
+        print "Got error: #{e}\n"
+        # TODO: catch error
```

Comment:
Is this to be done prior to merge? Or is it some future work needed?

Reply:
Done now